java/src/main/native/ai_onnxruntime_OrtSession_SessionOptions.c

```diff
@@ -639,33 +637,29 @@ JNIEXPORT void JNICALL Java_ai_onnxruntime_OrtSession_00024SessionOptions_addExe
   const char* epName = (*jniEnv)->GetStringUTFChars(jniEnv, jepName, NULL);
   const OrtApi* api = (const OrtApi*)apiHandle;
   OrtSessionOptions* options = (OrtSessionOptions*)optionsHandle;
-  size_t keyCount = (*jniEnv)->GetArrayLength(jniEnv, configKeyArr);
+  int keyCount = (*jniEnv)->GetArrayLength(jniEnv, configKeyArr);
```

Comment:
The return type of GetArrayLength is jsize, which is an alias of jint, which is `int`. We should  avoid changing it to an unsigned type unless necessary.